README.md

```diff
@@ -2102,6 +2103,53 @@ The error message thrown by mailgun is passed to the response object when an ema
 Mailgun error codes can be found
 [here](https://documentation.mailgun.com/en/latest/api-sending.html#error-codes).
 
+## Localization
+
+### Setting application properties
+
+Form flow library will read the languages set in an apps `application.properties`.
+
+```yaml
+# application.yaml
+form-flow:
+  languages: en, es
+```
+
+Our language selector menu button will create links both inside of the button and nav links based on
+the value provided in `form-flow.languages`. If none is given, the default is: `en`.
+
+#### Language tag syntax
+
+Refer to
+the [language tag syntax](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/lang#language_tag_syntax)
+to see what language you want to support.
+
+### Supplying translation files
+
+Translations need to be added to `messages_xx.properties` files in the `src/main/resources`
```

Comment:
It might be good to specify/add `in the applications 'src/main/resources directory'`, so any reader realizes the configuration of this is all in their app and not the library.